src/app/pages/plugins/plugins.component.ts

```diff
@@ -57,7 +57,7 @@ export class PluginsComponent {
       doubleConfirm: (item) => {
         return this.dialogService.doubleConfirm(
           T('Verify Deletion of ') + item.name + T(' Plugin'),
-          T('To delete the <b>') + item.name + T('</b> plugin and all data and snapshots stored with it, please type the name of the plugin to confirm:'),
+          T('To delete the <b>') + item.name + T('</b> plugin and all data and snapshots stored with it, please type the name of the plugin to confirm.'),
```

Comment:
Can we make this a generic, reusable routine?  We already have similar functions here and for deleting datasets. We could reuse it for shares, users, groups, and other things.

Reply:
That's what this PR does. The above text is the specific instructions for deleting plugins. They probably should be changed, because the reusuable part appears below  this:

`Enter <plug-in name> below to confirm.`)

This works for datasets too, and for any other place we use the double confirm.

Reply:
PS - I updated the directions specific to plugins. 
![Screenshot from 2019-10-14 11-53-53](https://user-images.githubusercontent.com/9504493/66765362-c2e8c900-ee79-11e9-85a8-88471defb58e.png)
![Screenshot from 2019-10-14 11-54-30](https://user-images.githubusercontent.com/9504493/66765365-c5e3b980-ee79-11e9-9369-2947ea06f852.png)